### PR TITLE
chore: update app translations

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

Weekly translation flow by the Globalization Workstream.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk

None, only translations.

### CODE REVIEW

File: src/assets/translations/fr/main.json [hunk 1]
```diff
@@ -231,10 +246,11 @@
     "notSupported": "%{asset} n'est pas pris en charge par %{wallet}"
   },
   "connectWalletPage": {
+    "title": "L'échange multichaîne original",
     "shapeshift": "ShapeShift",
     "exploreThe": "Explorez",
     "defiUniverse": "l'Univers DeFi",
-    "body": "100% Gratuit. Open‑Source. Propriété communautaire. Non‑dépositaire. Privé.",
+    "body": "Échangez, déplacez et gagnez. Privé. Propriété communautaire. Non‑dépositaire. Décentralisé.",
```
Comment (gomesalexandre) [inline src/assets/translations/fr/main.json:253]: Not sure about the `bridge` translation here. Maybe let's keep it more English for this one instance i.e "bridgez"?

**Review by gomesalexandre — APPROVED** [review]
One comment WRT the French translation but LGTM

**Review by gomesalexandre — APPROVED** [review]
💜

File: src/assets/translations/fr/main.json (hunk 1 above)
Comment (firebomb1) [reply src/assets/translations/fr/main.json]: You're right it misses some of semantic. As discussed in Discord, we've settled to use a neologism/frenglish "Bridger" for now and "Transfer/Transfert" when context is present. I've updated the translations across the app based on this. Thanks for the review !

**Pull request merged**